src/backends/perf_pt/mod.rs

```diff
@@ -616,6 +612,22 @@ mod tests {
         (ptxed_args, vdso_tempfile)
     }
 
+    /*
+     * Determine if the given x86_64 assembler mnemonic should terminate a block.
+     *
+     * Mnemonic assumed to be lower case.
+     */
+    fn instr_terminates_block(instr: &str) -> bool {
+        assert!(!instr.starts_with("["));
+        match instr {
+            // JMP or Jcc are the only instructions beginning with 'j'.
+            m if m.starts_with("j") => true,
```

Comment:
Let me point out here that `Jcc` is not one instruction. `cc` is a place-holder for a condition, so there's `jg`, `jge`, `jz`... Nearly a hundred variations in total. That's why I didn't enumerate them all.

Reply:
Sorry, that's nearly a hundred *encodings* (depending on the types of the operands). So we'd have to list less, looks like 35 or so, if we didn't wan't to use this catch-all case.